src/test/java/com/epam/tc/hw2/ExerciseOneTest.java

```diff
@@ -0,0 +1,83 @@
+package com.epam.tc.hw2;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class ExerciseOneTest extends AbstractTest {
+
+
+    private static String XPATH_ITEMS_IN_HEADER = " //ul[contains(@class,'uui-navigation nav ')]/li";
+    private static String CSS_LEFT_MENU = ".sidebar-menu > li";
+    private static String ID_FRAME_BUTTON = "frame-button";
+    private static List<String> LIST_ITEMS_HEADER =  new ArrayList<>(List
+            .of("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"));
+    private static List<String> LIST_ITEMS_LEFT_MENU =  new ArrayList<>(List
+            .of("Home", "Contact form", "Service", "Metals & Colors", "Elements packs"));
```

Comment:
Please, check how to use `List.of()` Method [here](https://www.softwaretestinghelp.com/java-list-how-to-create-initialize-use-list-in-java/)